auth/auth-application/src/main/java/me/nalab/auth/application/service/AuthTokenCreateService.java

```diff
@@ -0,0 +1,38 @@
+package me.nalab.auth.application.service;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import me.nalab.auth.application.common.dto.AuthToken;
+import me.nalab.auth.application.common.dto.CreateAuthTokenRequest;
+import me.nalab.auth.application.common.dto.Payload;
+import me.nalab.auth.application.common.utils.JwtUtils;
+import me.nalab.auth.application.port.in.web.AuthTokenCreateUseCase;
+
+@Service
+@RequiredArgsConstructor
+public class AuthTokenCreateService implements AuthTokenCreateUseCase {
+
+	private final JwtUtils jwtUtils;
+
+	@Override
+	public AuthToken create(CreateAuthTokenRequest request) {
+		var userId = request.getUserId();
+		var nickname = request.getNickname();
+		if (userId.isBlank() || nickname.isBlank()) throw new IllegalArgumentException();
```

Comment:
저희 컨벤션이 아래처럼 되어있던거 같아요!
```suggestion
		if (userId.isBlank() || nickname.isBlank()) {
			throw new IllegalArgumentException();
		}
```